src/components/SearchBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import '../styles/search.scss'
+
+export const SearchBar = () => {
```

Comment:
You sure this is a `const`? I tried including a `<SearchBar>` element and get an error.

`throw new Error('Element type is invalid: expected a string (for built-in ' + 'components) or a class/function (for composite components) ' + ("but got: " + (type == null ? type : typeof type) + "." + info));`

If I change this line to this:

`export default function SearchBar({ children }) {`

then it works. Am I including it wrong somehow?